### DISCUSSION TITLE
Nullary UFs possible?

### DISCUSSION THREAD

**Discussion by baierd** [cvc5/cvc5#8868] [Q&A] [answered]
Hello,

is it possible to create nullary UFs in CVC5?
I tried `mkFunctionSort()` with an empty list of sorts as arguments and also a `NullSort` with no luck.
I am using the Java bindings if that changes something.

Thank you for your help!

**Comment by 4tXJ7f** [comment, marked as answer]
Hi @baierd, sorry about the late reply. [`Solver.mkConst()`](https://cvc5.github.io/docs/cvc5-1.0.0/api/java/io/github/cvc5/Solver.html#mkConst(io.github.cvc5.Sort,java.lang.String)) should do the trick. It corresponds to `(declare-fun <symbol> () <sort>)` in SMT-LIB.
